how-to/register-with-browser/client/src/launchbar.ts

```diff
@@ -0,0 +1,121 @@
+import { fin } from "openfin-adapter/src/mock";
+import { BrowserCreateWindowRequest, BrowserButtonType, BrowserWindowModule, getCurrentSync, Page, ToolbarOptions, WorkspacePlatformModule, PageLayout } from '@openfin/workspace-platform';
+import { createPageWithLayout, createViewIdentity } from "./browser";
+
+const platform: WorkspacePlatformModule = getCurrentSync();
+const defaultPageLayout: PageLayout = {
+    content: [
+        {
+            type: 'stack',
+            content: [
+                {
+                    type: 'component',
+                    componentName: 'view',
+                    componentState: {
+                        identity: createViewIdentity(fin.me.uuid, 'v1'),
+                        url: 'https://examples.com'
+                    }
+                },
+                {
+                    type: 'component',
+                    componentName: 'view',
+                    componentState: {
+                        identity: createViewIdentity(fin.me.uuid, 'v2'),

```

Comment:
question (because I missed it...are we creating a view identity because we use that somewhere else? Shouldn't workspace generate an id if it isn't specified.